packages/google_sign_in/lib/google_sign_in_tizen.dart

```diff
@@ -84,28 +82,31 @@ class _Credentials {
 
 class _CachedTokenStorage {
   // ignore: invalid_use_of_visible_for_testing_member
-  final SecureStorage _storage = SecureStorage();
+  final FlutterSecureStorage _storage = const FlutterSecureStorage();
 
   final String _kToken = 'token';
 
   /// Cached token.
   _GoogleSignInTokenDataTizen? _token;
 
   Future<void> saveToken(_GoogleSignInTokenDataTizen token) async {
-    await _storage.saveJson(_kToken, token.toJson());
+    await _storage.write(key: _kToken, value: jsonEncode(token.toJson()));
     _token = token;
   }
 
   Future<_GoogleSignInTokenDataTizen?> getToken() async {
     if (_token != null) {
       return _token!;
     }
-    final Map<String, Object?>? json = await _storage.getJson(_kToken);
-    return json != null ? _GoogleSignInTokenDataTizen.fromJson(json) : null;
+    final String? jsonString = await _storage.read(key: _kToken);
+    return json != null
```

Comment:
`json` is in `dart:convert`.

```suggestion
    return jsonString != null
```